deps/build.jl

```diff
@@ -2,24 +2,29 @@ using BinDeps
 
 @BinDeps.setup
 
-zstd = library_dependency("zstd", aliases=["libzstd", "libzstd.1"])
+vers = v"1.0.0"
 
-if is_apple()
-    using Homebrew
-    provides(Homebrew.HB, "zstd", zstd, os=:Darwin)
-else
-    if isdir(srcdir(zstd))
-        rm(srcdir(zstd), recursive=true)
-        mkdir(srcdir(zstd))
-    end
+zstd = library_dependency("zstd", aliases=["libzstd", "libzstd.$(vers.major)", "libzstd.$vers"])
 
-    if isdir(BinDeps.downloadsdir(zstd))
-        rm(BinDeps.downloadsdir(zstd), recursive=true)
-        mkdir(BinDeps.downloadsdir(zstd))
+function cleardir(d)
+    if isdir(d)
+        rm(d, recursive=true)
+        mkdir(d)
     end
+    return
+end
 
-    vers = v"1.0.0"
+cleardir(srcdir(zstd))
+cleardir(BinDeps.downloadsdir(zstd))
+cleardir(libdir(zstd))
 
+if is_apple()
+    using Homebrew
+    provides(Homebrew.HB, "zstd", zstd, os=:Darwin)
+elseif is_windows()
+    dlurl = "https://dl.bintray.com/ararslan/generic/libzstd-$(Sys.WORD_SIZE).zip"
+    provides(Binaries, URI(dlurl), zstd, unpacked_dir=libdir(zstd), os=:Windows)
```

Comment:
should make a `usr/bin` in the zip files and put the dll there, then you shouldn't need to specify `unpacked_dir`

from what I remember that last time I tried setting `unpacked_dir = "."` or otherwise empty it didn't go well